filesystem/src/main/java/org/frankframework/filesystem/AbstractFileSystemListener.java

```diff
@@ -140,13 +141,22 @@ public void configure() throws ConfigurationException {
 		if (getNumberOfBackups()>0 && !(fileSystem instanceof IWritableFileSystem)) {
 			throw new ConfigurationException("FileSystem ["+ClassUtils.nameOf(fileSystem)+"] does not support setting attribute 'numberOfBackups'");
 		}
+		if (isFileTimeSensitive() && !(fileSystem instanceof IWritableFileSystem)) {
+			throw new ConfigurationException("FileSystem ["+ClassUtils.nameOf(fileSystem)+"] does not support setting attribute 'fileTimeSensitive'");
+		}
 		knownProcessStates = ProcessState.getMandatoryKnownStates();
 		for (ProcessState state: ProcessState.values()) {
 			if (StringUtils.isNotEmpty(getStateFolder(state))) {
 				knownProcessStates.add(state);
 			}
 		}
 		targetProcessStates = ProcessState.getTargetProcessStates(knownProcessStates);
+		if (!(knownProcessStates.contains(ProcessState.INPROCESS) && isFileTimeSensitive()) && !(isOverwrite() || getNumberOfBackups() > 0)) {
+			ConfigurationWarnings.add(this, log, "It is recommended to configure either an in-process folder and to set 'fileTimeSensitive', or configure 'overwrite' or 'numberOfBackups' to avoid problems when files with the same name are processed.");
```

Comment:
Things I discovered while debugging the code... Knowledge best captured somewhere.

Reply:
Als ik dat op de AbstractFileSystemListener zet, wordt dat dan ook automatisch meegenomen in de gegenereerde Frank!Doc van de subclasses of moet ik daar ook nog wat aan toevoegen?

Reply:
Het wordt geinherit

Reply:
Niet vanzelf geloof ik, dus ik heb `@inheritDoc` annotaties toegevoegd.